webcrypto/test_setup.go

```diff
@@ -54,51 +35,14 @@ func newTestSetup(t testing.TB) testSetup {
 	// We execute the helpers script in the goja runtime
 	// in order to make the Web Platform helpers available
 	// to the tests.
-	_, err = rt.RunProgram(helpersProgram)
+	_, err = ts.VU.Runtime().RunProgram(helpersProgram)
 	require.NoError(t, err)
 
-	root, err := lib.NewGroup("", nil)
-	require.NoError(t, err)
-
-	samples := make(chan metrics.SampleContainer, 1000)
-
-	state := &lib.State{
-		Group:  root,
-		Dialer: tb.Dialer,
-		Options: lib.Options{
-			SystemTags: metrics.NewSystemTagSet(
-				metrics.TagURL,
-				metrics.TagProto,
-				metrics.TagStatus,
-				metrics.TagSubproto,
-			),
-			UserAgent: null.StringFrom("TestUserAgent"),
-		},
-		Samples:        samples,
-		TLSConfig:      tb.TLSClientConfig,
-		BuiltinMetrics: metrics.RegisterBuiltinMetrics(metrics.NewRegistry()),
-		Tags:           lib.NewVUStateTags(metrics.NewRegistry().RootTagSet()),
-	}
-
-	vu := &modulestest.VU{
-		CtxField:     tb.Context,
-		InitEnvField: &common.InitEnvironment{},
-		RuntimeField: rt,
-		StateField:   state,
-	}
+	m := new(RootModule).NewModuleInstance(ts.VU)
+	require.NoError(t, ts.VU.Runtime().Set("crypto", m.Exports().Named["crypto"]))
+	require.NoError(t, ts.VU.Runtime().GlobalObject().Set("CryptoKey", CryptoKey{}))
```

Comment:
you might want to make it also use the module system and then use a simple script to make it globally accessible as in 

https://github.com/grafana/k6/blob/f38347d45bceb8bb42bfbd8785d1d73940aa8b6b/js/modules/k6/experimental/fs/module_test.go#L684-L706